backend/src/routes/api/gpu/gpuUtils.ts

```diff
@@ -1,9 +1,19 @@
 import { KubeFastifyInstance, PrometheusResponse } from '../../../types';
 import { V1PodList } from '@kubernetes/client-node';
 import https from 'https';
-import * as fs from 'fs';
+
+/** Storage to prevent heavy calls from being performed for EVERY user */
+const storage: { lastFetch: number; lastValue: [boolean, number] } = {
+  lastValue: [false, 0],
+  lastFetch: 0,
+};
 
 export const getGPUNumber = async (fastify: KubeFastifyInstance): Promise<[boolean, number]> => {
+  if (storage.lastFetch >= Date.now() - 30_000) {
```

Comment:
Should 30_000 be a constant?

Reply:
Ah probably, I did this -- I was intended to slow down the costly effort of determining GPUs.

Reply:
so just `const delay = 30000` ?

Reply:
Might be better if we named it something like `CACHE_DELAY` and always please keep the `_` when dealing with 5+ digits in numbers.